### ISSUE TITLE
Disabled extension youtube: Dependency The 'bs4' distribution was not found and is required by the application not found

### ISSUE THREAD

**Issue by solsticedhiver** [natumbri/mopidy-youtube#137]
When running mopidy with mopidy-youtube extension installed globally (not with pip but setup.py), mopidy gives this error, even though beatuifulsoup 4 is installed

    Disabled extension youtube: Dependency The 'bs4' distribution was not found and is required by the application not found
    [...]
    Disabled extensions: youtube

**Comment by natumbri** [comment]
Thanks @solsticedhiver - I'm not sure I understand how you have installed mopidy-youtube and I certainly haven't tested installing it that way. 
Can you explain further what you've done (and maybe why) and I'll see if I can track down the bug?

**Comment by solsticedhiver** [comment]
I have made a package with _makepkg_ and a **PKGBUILD** on _archlinux-arm_. using `setup.py`

So basically this is just running setup.py install with --root=$pkgdir

You setup.py is quite empty. I doubt it could work like that.

I just tried running as root setup.py install, and I noticed it download and install some eggs. I don't except that to happen.

My guess would be that your extension except to find bs4 egg but does not look for the beatifulsoup globally installed
